Index: linux-6.6.100/drivers/staging/fsl_ppfe/pfe_eth.c
===================================================================
--- a/drivers/staging/fsl_ppfe/pfe_eth.c
+++ b/drivers/staging/fsl_ppfe/pfe_eth.c
@@ -2130,7 +2130,7 @@ static struct sk_buff *pfe_eth_rx_skb(st
 
 		/* First frag */
 		if (desc_ctrl & CL_DESC_FIRST) {
-			skb = build_skb(buf_addr, 0);
+			skb = slab_build_skb(buf_addr);
 			if (unlikely(!skb))
 				goto pkt_drop;
 
@@ -2152,7 +2152,7 @@ static struct sk_buff *pfe_eth_rx_skb(st
 				goto pkt_drop;
 			}
 
-			skb_frag = build_skb(buf_addr, 0);
+			skb_frag = slab_build_skb(buf_addr);
 
 			if (unlikely(!skb_frag)) {
 				kfree(buf_addr);
